### DISCUSSION TITLE
Allow linking to a draft issue

### DISCUSSION THREAD

**Discussion by renegalindo** [community/community#10903] [Projects and Issues] [answered]
Current draft issues do not have a url of their own so it's impossible to share a link directly to them. You can only link to the whole project page. 

I understand they are meant to be "drafts" and eventually be turned into issues, but this is such a basic feature that should definitively be there.

**Comment by Fiech** [comment]
Yup, especially because you can already assign them within your organization!

**Comment by bplatta** [comment]
This is the main pain point for my org in using the new projects. I'd like to be able to send an engineer a url to specific draft

**Comment by chrisglein** [comment]
Duplicates:
https://github.com/github-community/community/discussions/16610
https://github.com/github-community/community/discussions/15611

**Comment by rileybroughten** [comment, marked as answer]
Linking to draft items within a project is now supported, check out the announcement [here](https://github.blog/changelog/2023-01-05-github-issues-january-5th-update/#%F0%9F%94%97-automatically-open-an-item-in-the-side-panel-via-deep-link)!

**Comment by fiedl** [comment]
Is it already possible to retrieve this url via the [graphql api](https://docs.github.com/en/graphql/overview/about-the-graphql-api)?

**Comment by lisastaehli** [comment]
Where can I find information about when this will be available in GitHub Enterprise :)?

**Reply by rileybroughten** [reply]
If you're referring to copying the link to a draft item in a project, you can see this supported in GHES 3.9 and above

**Comment by Malix-Labs** [comment]
See #108378 that would make this be a native issue's object feature
